module/VuFind/src/VuFind/Controller/AlphabrowseController.php

```diff
@@ -185,6 +190,24 @@ protected function addResultsToView(
         }
     }
 
+    /**
+     * Applies topic delimiters to the 'heading' field of each item in the browse results.
+     *
+     * @param array &$result The result array containing 'Browse' items to be modified.
+     *
+     * @return void
+     */
+    protected function applyTopicDelimiters(&$result): void
+    {
+        $config = $this->getConfig();
+
+        foreach ($result['Browse']['items'] as &$item) {
+            $item['heading'] = str_replace("\u{2002}",
+                                           ($config->AlphaBrowse->topic_browse_separator ?? ' > '),
+                                           $item['heading']);
```

Comment:
Suggested style change for consistency:

```suggestion
            $item['heading'] = str_replace(
                "\u{2002}",
                ($config->AlphaBrowse->topic_browse_separator ?? ' > '),
                $item['heading']
            );
```

Reply:
Done!